### ISSUE TITLE
Can 2.0.2 be released?

### ISSUE THREAD

**Issue by nhoffmann** [redis-store/redis-rack-cache#22]
The currently released version locks `redis-store` to a version that has a known security vulnerability ([CVE-2017-1000248](https://nvd.nist.gov/vuln/detail/CVE-2017-1000248)). 

Version `2.0.2` relaxes the `redis-store` dependency. Can this version be released? Can we do anything to help?

**Comment by openfirmware** [comment]
I'm working on the updating dependencies for a Rails app and until 2.0.2 is released, I was able to do this in the Gemfile to get the latest commit:

```ruby
gem 'redis-rack-cache',
  git: 'https://github.com/redis-store/redis-rack-cache.git',
  ref: 'd1fdd3868bdc02efa5ae724a2758b3abfda271ab'
```

(Only a temporary solution)

**Comment by tilo** [comment]
bump!

**Comment by tubbo** [comment]
hey folks, we just released **v2.0.2**. apologies for the wait!
